static/js/src/dynamic-forms.js

```diff
@@ -505,6 +508,24 @@
 
       setCheckboxLimit();
 
+      // Setup dial code dropdown options (intlTelInput)
+      function setupIntlTelInput() {
+        const utilsScript =
+          "https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/17.0.18/js/utils.js";
+
+        // remove name from original input so only the hidden input is submitted
+        const inputName = phoneInput.name;
+        phoneInput.removeAttribute("name");
+
+        intlTelInput(phoneInput, {
+          utilsScript,
+          separateDialCode: true,
+          hiddenInput: inputName,
+        });
+      }
+
+      setupIntlTelInput();
+
```

Comment:
It looks like it's not necessary to repeat the script like this, both dynamic-form.js and intTelInput.js are included as part of main.js - or is this happening because dynamic-form.js needs to do a lot of work only when the CTA is clicked?

If that's the case, you could add this script to the utils directory and export it as a module (see debounce.js), and just call the imported function in this script to reduce the repetition. To have it still initialise on page load for static forms, I think you could just call the exported function in the util file, and make sure it's included in main.js (happy to talk through this).

Reply:
I don't think that will work as the modal needs to be initialized before calling intlTelInput, as it needs to be passed the `input#phone`. Which is not available until you click the CTA.

Reply:
That's what I mean - you would import the function, and then call it here when it's appropriate.

Reply:
Ok, I think I understand. Can you take another look when you have a moment please